woof-distro/x86_64/void/voidpup64/DISTRO_PET_REPOS

```diff
@@ -0,0 +1,63 @@
+#------------------
+#PKG_DOCS_PET_REPOS - where to download the pet pkgs databases from.
+#------------------
+# 1|2|3
+#   1 - domain. for testing the url.
+#   2 - full URI of the database file.
+#   3 - name of db file when local and after being processed into standard format
+#  (in the case of PET databases, the names are the same and no processing is required)
+
+if ( echo "$BUILD_FROM_WOOF" | grep -q ';' ) ; then
+	WCE_BRANCH=$(echo "$BUILD_FROM_WOOF" | cut -f 1 -d ';')
+else
+	WCE_BRANCH=testing
+fi
+
+PKG_DOCS_PET_REPOS="
+z|http://smokey01.com/peebee/voidpup/Packages-puppy-vpup64-extra|z
+z|https://github.com/puppylinux-woof-CE/woof-CE/raw/testing/woof-distro/x86_64/slackware64/14.2/Packages-puppy-slacko6414.2-official|z
+z|https://github.com/puppylinux-woof-CE/woof-CE/raw/testing/woof-distro/x86_64/Packages-puppy-common64-official|z
+z|https://github.com/puppylinux-woof-CE/woof-CE/raw/testing/woof-distro/Packages-puppy-noarch-official|z
+"
+
+#---------
+#PET_REPOS - hardcode the pet repos in here...
+#---------
+# 1|2|3
+#   1 - domain. for testing the url.
+#   2 - full URI of the repo
+#   3 - name of db-file(s) associated with that repo. it may have glob wildcards.
+#   ex: Packages-puppy-4-official (note, url paths are in the database)
+
+PET_REPOS='
+z|http://smokey01.com/peebee/voidpup|Packages-puppy-vpup64-extra
```

Comment:
`http://smokey01.com/peebee/voidpup/pet_packages-vpup64` doesn't exist

Reply:
I'll fix - the 32-bit does exist....